sql/core/src/test/java/test/org/apache/spark/sql/JavaDataFrameSuite.java

```diff
@@ -24,6 +24,7 @@
 import java.math.BigInteger;
 import java.math.BigDecimal;
 
+import org.apache.spark.sql.*;
```

Comment:
??? - Is this passing style checks?

Reply:
it did, but I'll reset it, intellij being helpful.  I'll re-fix that.